autopush/router/fcm.py

```diff
@@ -50,6 +50,8 @@ def register(self, uaid, router_data, router_token=None, *kwargs):
         if "token" not in router_data:
             raise self._error("connect info missing FCM Instance 'token'",
                               status=401)
+        if not (router_token == router_data['token'] == self.senderID):
```

Comment:
You may or may not want to copy the associated comment over from gcm.py, up to you